### PR TITLE
Add logrotate sidecar

### PR DESCRIPTION
**Pull request by edvgui**
- Add logrotate sidecar
- Drop static ip configuration (not required)
- Add more env var to customize the behavior
- Document composition  

Part of https://github.com/inmanta/inmanta-core/issues/7452

### CODE REVIEW

**Review by wouterdb — COMMENTED** [review]
Are these override files used commonly?

I.e. this has to be maximally understandable. 
(I like it)

**Comment by edvgui** [comment]
> Are these override files used commonly?
> 
> I.e. this has to be maximally understandable. (I like it)

It is part of the documented way to "merge" docker compose files: https://docs.docker.com/compose/how-tos/multiple-compose-files/merge/  However, it seems I have mixed it up a bit, the override keyword means that docker-compose will automatically load this file when loading the similarly named file (without the override suffix).  So here, adding a simple extension seems to match the documentation better (I will update it).

File: README.md [hunk 1]
```diff
@@ -1,92 +1,151 @@
...
-sudo docker compose down
-
-# Clear storage or db and orchestrator
-sudo docker compose down -v
+sudo docker compose \
+    -f docker-compose.yml \
+    [-f docker-compose.iso.yml] \ # Deploy service orchestrator instead of oss one
+    [-f docker-compose.ssh.yml] \ # Deploy an ssh sidecar to access the orchestrator file system via ssh
+    [-f docker-compose.logrotate.yml] \ # Deploy a logrotate sidecar to rotate the logs of the orchestrator
+    <up|down|ps> [options...]
```
Comment (wouterdb) [inline README.md:36]: so nice

File: README.md [hunk 2]
```diff
@@ -1,92 +1,151 @@
...
+sudo docker compose -f docker-compose.yml ps -a
 
-# Stop db, orchestrator and ssh sidecar
-sudo docker compose -f docker-compose.yml -f docker.compose.ssh.override.yml down
+# Stop db and orchestrator
+sudo docker compose -f docker-compose.yml down
 
 # Clear storage or db and orchestrator
-sudo docker compose -f docker-compose.yml -f docker.compose.ssh.override.yml down -v
+sudo docker compose -f docker-compose.yml down -v
```
Comment (wouterdb) [inline README.md:57]: I wonder if we should make these individual code blocks to make clear they are not one sequence .

(and with that I mean, I really don't know what is best, just remarking it)

**Review by wouterdb — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (edvgui) [reply README.md:57]: I think I will keep it as it is for now, I like that it shows the shared context of the commands (the exported env vars are required for all commands, if we split them in different code blocks, we would need to provide them for every command block)

**Pull request merged**